### DISCUSSION TITLE
Role Reporter

### DISCUSSION THREAD

**Discussion by geoplanteam** [opengisch/QField#2465] [Q&A]
Dear QField developper team

Thanks for all your great work!
I have questions about the available roles in QField.
If my understandig ist right, then

- the role reader is just for viewing. No editing is possible. 
- the role reporter is to map, edit and delete my data. Data of others is just for viewing.
- the role editor ist to map, edit and delete data of everybody.
- the role manager has no other rights in the app, but in the backend for configuration.
- the role admin has no other rights in the app, but all rights for configuration in the backen.

I encountered problems with the very usefull role of reporter. The modified data wasn't pushed to the cloud (push changes). And when I use synchronize, my local data is lost.

Am I doing something wrong or is this an issue?

Regards Christoph

**Comment by suricactus** [comment]
Hello, @geoplanteam . Thanks for trying QFieldCloud.

I will have a look and share results in the coming weeks.

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).

**Comment by ZsanettMed** [comment]
Dear geoplanteam, sorry for the delayed answer! We are improving the documentation, which will include a better description of the available roles. But we also hugely improved the application. Can you try it with the latest version and give a feedback if the problem still exists?

**Comment by geoplanteam** [comment]
Hello @suricactus 
I'm sorry, I still have the same problem. Pushes from the role reporter are not taken into account. And the synchronization erases all the data I have captured on the mobile/tablet.

But I have an other issue when I thry to synchronize my project in QGIS: Upload from local to cloud (prefere local) works fine. Download from cloud (prefere cloud) doesn't work. The message is: Aborted "instandhaltung.gpkg" download.

If you don't have any of this problems, I will try with a new, clean project.

Regards Christoph

**Reply by aliasatp** [reply]
Hi @geoplanteam , @suricactus 
have you tested the reporter role? I tried it but all modifications I pushed on the cloud are "classified" as unpermitted. Is there any documentation about the limitations of roles? I think it is very important that users under "manager role" shouldn't have the possbility to have a local data copy back in their pc (sync back to local). My idea. Thanks

**Reply by geoplanteam** [reply]
Hi @aliasatp 
I have tested all the roles. But with the reporter role I have problems to write data back to the cloud.
What is expected for every role is an other question. I wrote down my understanding of the roles in the very first message of this post.
Regards Christoph

**Reply by suricactus** [reply]
I will look into your issue next week and the documentation is already under ongoing improvement on roles. I will keep you posted.
